### PR TITLE
PumpDanaR unit tests

### PR DESCRIPTION
**Pull request by PoweRGbg**
finally in one commit

### CODE REVIEW

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/comm/MsgStatusBolusExtended.java [hunk 1]
```diff
@@ -68,10 +68,6 @@ private long getDateFromSecAgo(int tempBasalAgoSecs) {
 
     public static void updateExtendedBolusInDB() {
         TreatmentsInterface treatmentsInterface = new TreatmentsPlugin();
-        if ( TreatmentsPlugin.getPlugin() != null) {
-            treatmentsInterface = TreatmentsPlugin.getPlugin();
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/comm/MsgStatusBolusExtended.java]: 'treatmentsInterface = TreatmentsPlugin.getPlugin();' this is correct

File: app/src/test/java/info/AAPSMocker.java [hunk 2]
```diff
@@ -113,7 +113,6 @@ public static MainApp mockMainApp() {
         MainApp mainApp = mock(MainApp.class);
         DanaRPlugin danaRPlugin = mock(DanaRPlugin.class);
```
Comment (MilosKozak) [inline app/src/test/java/info/AAPSMocker.java]: this as well

**Comment by MilosKozak** [comment]
merge current dev to see if test pass

**Comment by PoweRGbg** [comment]
Travis fails, Local build is ok and tests ok

**Comment by MilosKozak** [comment]
ok, looks good now but it's reverting changes dev (check the diff here)
i believe it's necessary to create new branch and PR, only with wanted changes

**Comment by PoweRGbg** [comment]
Closing for another try :)

**Pull request closed without merging**